src/vivarium_public_health/metrics/disability.py

```diff
@@ -43,7 +38,7 @@ class DisabilityObserver:
                         - "sample_stratification"
     """
 
-    configuration_defaults = {
+    CONFIGURATION_DEFAULTS = {
```

Comment:
Why all caps for this? Because it's intended as a constant?

Reply:
Yes, exactly.